books/english-composition/shorten

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '9.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :english_composition_short) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  # trash all but first chapter
+  book.chapters.drop(2).each(&:trash)
+  # trash all but first appendix
+  book.search('div.appendix').drop(1).each(&:trash)
```

Comment:
Should chapters drop 2 but appendix only drop 1?

Reply:
@LostMojo There can be chapter.drop(2) and appendix.drop(1) in the same shorten file. It f.ex. in sociology recipe. But I changed it anyway because now there is only one chapter with fix content available in tests. I also added units. :slightly_smiling_face: